packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -263,7 +264,7 @@ void main() {
         expect(fakeVideoPlayerPlatform.dataSources[0].package, null);
       });
 
-      test('network', () async {
+      test('legacy network', () async {
```

Comment:
Rather than interleaving the legacy and non-legacy versions, please make a new `group` called `legacy` and put the legacy tests in that group.